### PR TITLE
Extract chat component out of playground

### PR DESCRIPTION
**Pull request by Henry-Pulver**
This has changed the aesthetic of the playground slightly :smile: 

Screenshot from 2023-09-02 22-28-29

### CODE REVIEW

File: components/playground.tsx [hunk 1]
```diff
@@ -80,16 +151,17 @@ export default function Playground() {
               apis.some((api) => !api.api_host) &&
               !mockApiResponses)
               ? `You need to add${numActions ? "" : " actions (Actions tab)"}${
-                  numActions ||
-                  (apis?.length && apis.some((api) => !api.api_host)) ||
-                  mockApiResponses
+                  !numActions &&
+                  ((apis?.length && apis.some((api) => !api.api_host)) ||
+                    !mockApiResponses)
```
Comment (neuromantic99) [inline components/playground.tsx:156]: something up with the string here:

image

Comment (neuromantic99) [reply components/playground.tsx:156]: I noticed we aren't checking for an api key, is it possible to have an API that doesn't require a key?

File: components/playgroundChatbot.tsx [hunk 2]
```diff
@@ -62,205 +51,20 @@ export default function PlaygroundChatbot(props: {
...
-      setUsageLevel((prev) => prev + 1);
-      if (loading || alreadyRunning.current) return;
-      alreadyRunning.current = true;
-      setLoading(true);
-      const response = await fetch("/api/v1/answers", {
-        method: "POST",
-        headers: {
-          "Content-Type": "application/json",
-          Authorization: `Bearer ${profile!.organizations!.api_key}`,
-          sessionToken: session?.access_token ?? "",
```
Comment (Henry-Pulver) [inline components/playgroundChatbot.tsx:119]: Removal of this means you need to use the `cookie` in `/answers` to get the session token now :smile:

**Review by Henry-Pulver — COMMENTED** [review]
The playground is now too wide - previous dashboard width was better

**Pull request merged**